### PR TITLE
Bbgw support

### PR DESCRIPTION
**Pull request by telphan**
@agherzan @floion @petrosagg

### CODE REVIEW

File: layers/meta-resin [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 94f33cbb3b0ddfa3b7f91edb23716893cbbf763e
```
Comment (agherzan) [inline layers/meta-resin]: Please use "meta-resin: Use version 1.14" commit log format.

File: layers/meta-resin-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging/0003-am335x_evm.h-Change-default-dtb-for-Beaglebone-Green-Wireless.patch [hunk 2]
```diff
@@ -0,0 +1,26 @@
+From c55e9cb4d0fd5ebbbc2389b85c12e2f21fc775dd Mon Sep 17 00:00:00 2001
+From: Theodor Gherzan <theodor@resin.io>
+Date: Wed, 18 May 2016 14:43:57 +0200
+Subject: [PATCH] am33x_evm.h: Change default dtb for the Beaglebone Green Wireless
+
+Signed-off-by: Theodor Gherzan <theodor@resin.io>
```
Comment (agherzan) [inline layers/meta-resin-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging/0003-am335x_evm.h-Change-default-dtb-for-Beaglebone-Green-Wireless.patch]: Upstream status = configuration

File: beaglebone-green-wifi.coffee [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+postProvisioningInstructions = [
+	instructions.BOARD_SHUTDOWN
+	instructions.REMOVE_INSTALL_MEDIA
+	instructions.BOARD_REPOWER
+]
+
+module.exports =
+	slug: 'beaglebone-green-wifi'
+	aliases: [ 'beaglebone-green-wifi' ]
```
Comment (agherzan) [inline beaglebone-green-wifi.coffee]: remove as it is the same as slug

File: beaglebone-green-wifi.coffee [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+	instructions: [
+		instructions.ETCHER_SD
+		instructions.EJECT_SD
+		instructions.FLASHER_WARNING
+		BBB_FLASH
+	].concat(postProvisioningInstructions)
+
+	gettingStartedLink:
+		windows: 'http://docs.resin.io/#/pages/installing/gettingStarted-BBB.md#windows'
```
Comment (agherzan) [inline beaglebone-green-wifi.coffee]: Are we going to have the same docs pages as for BBB? Check with Shaun please.

File: layers/meta-resin-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging/0003-am335x_evm.h-Change-default-dtb-for-Beaglebone-Green-Wireless.patch (hunk 2 above)
Comment (telphan) [inline layers/meta-resin-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging/0003-am335x_evm.h-Change-default-dtb-for-Beaglebone-Green-Wireless.patch]: Fixed

File: beaglebone-green-wifi.coffee (hunk 3 above)
Comment (telphan) [inline beaglebone-green-wifi.coffee]: Fixed

File: beaglebone-green-wifi.coffee (hunk 4 above)
Comment (telphan) [inline beaglebone-green-wifi.coffee]: Yes.

**Comment by floion** [comment]
LGTM

**Comment by agherzan** [comment]
LGTM

**Pull request merged**